src/components/molecules/CheckedResourcesActionsMenu/CheckedResourcesActionsMenu.tsx

```diff
@@ -46,15 +54,23 @@ const StyledMenu = styled(Menu)`
   }
 `;
 
-const deleteCheckedResourcesWithConfirm = (checkedResources: K8sResource[]) => {
+const deleteCheckedResourcesWithConfirm = (checkedResources: K8sResource[], dispatch: AppDispatch) => {
   let title = `Are you sure you want to delete the selected resources (${checkedResources.length}) ?`;
 
   Modal.confirm({
     title,
     icon: <ExclamationCircleOutlined />,
     centered: true,
     onOk() {
+      let alertMessage = '';
       return new Promise(resolve => {
+        checkedResources.forEach(resource => {
+          dispatch(removeResource(resource.id));
+          alertMessage += `${alertMessage && ' | '}${resource.name}\n`;
```

Comment:
shouldn't the template string be `${alertMessage} | ${resource.name}\n`?

Reply:
We want to the the delimiter ( `|` ) only if the message already contains a resource name.

Reply:
okay, I'll close this